YamlValidator/Schema.h

```diff
@@ -8,42 +8,19 @@
 #include <iostream>
 #include <fstream>
 #include <typeinfo>
+#include <format>
 
 #include "Types.h"
 #include "YamlParser.h"
 
 class Schema {
 public:
-	struct ValidationResult {
-		struct ValidationError {
-			enum SchemaError {
-				TypeMismatch,
-				UnexpectedValue,
-				UnknownError
-			};
-
-			/*int column;
-			int row;*/
-
-			std::variant<ParserError, SchemaError> error;
-
-			ValidationError(std::variant<ParserError, SchemaError> error/*, int column = -1, int row = -1*/) : error(error)/*, column(column), row(row)*/ {};
-		};
-
-		struct ValidationSuccess {
-
-		};
-
-		std::variant<ValidationError, ValidationSuccess> result;
-
-		ValidationResult(std::variant<ValidationError, ValidationSuccess> result) : result(result) { }
-	};
-
 	enum Types {
```

Comment:
Bör inte singular användas här?

```suggestion
	enum Type {
```